.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+indent_size = 2
```

Comment:
Won't that also end up including things like the README, LICENCE, .gitignore, etc that it shouldn't?